lib/utils/fetch.ts

```diff
@@ -18,17 +17,26 @@ async function http<T>(path: string, config?: RequestInit): Promise<T> {
     return response.json().catch(() => ({}));
 }
 
-export async function get<T>(path: string, config?: RequestInit): Promise<T> {
+export async function get<T>(path: string, params?: Record<string, string | number>, config?: RequestInit): Promise<T> {
+    const url = new URL(path);
+    url.search = new URLSearchParams(params as Record<string, string>).toString();
     const init = { method: 'get', ...config };
-    return await http<T>(path, init);
+    return await http<T>(url.href, init);
 }
 
-export async function post<T, U>(path: string, body: T, config?: RequestInit): Promise<U> {
+export async function post<T, U>(
+    path: string,
+    body: T,
+    params?: Record<string, string | number>,
+    config?: RequestInit,
+): Promise<U> {
```

Comment:
You could also spread the parameters on their respective rows like this in the other functions in the file too, to improve readability.

Reply:
The linter won't let me 😢